CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 - Bump coroutines to v1.6.3
 - Implement view effects to show overdue search results
 - Wire Mobius loop with overdue search screen UI
+- Don't show no patients found error view when search query is empty
+- Prefill search query when search history item is clicked
```

Comment:
We need to remove the extra change log here

Reply:
I think the dependency PR is merged in first